feat: support subgraph-level demand control (#8829)
apollographql/router: docs/source/routing/errors.mdx

The estimated cost of the query was greater than the configured maximum cost.

</Property>
<Property name="SUBGRAPH_COST_ESTIMATED_TOO_EXPENSIVE">

> The estimated cost of the query directed at a specific subgraph was greater than the configured maximum cost.

Reviewer: ```suggestion
The estimated cost of the query directed at a specific subgraph exceeds the configured maximum cost.
```
